lesson_20/homework20.py

```diff
@@ -0,0 +1,92 @@
+"""
+HW 20.1. SQL.
+
+1. Create a database for an online store with the following tables:
+- 'products': a table to store information about products,
+            including name, description, price, etc.
+- 'categories': a table for product categories.
+- the 'products' table should have a foreign key
+referencing the 'categories' table.
+
+2. Write an SQL script to create the specified tables.
+3. Add a few rows of data to each table.
+4. Execute a JOIN query that returns information
+about the products and their category names.
+"""
+
+import sqlite3
+
+conn = sqlite3.connect('online_store.db')
+cursor = conn.cursor()
+
+# Create a table 'categories'
```

Comment:
Please organize functions aka create_tables, insert_data, get_data and in main block call all these three functions + add  some basic error handling